grapple/types/images.py

```diff
@@ -59,6 +60,15 @@ def get_rendition_type():
     return rendition_type
 
 
+def rendition_allowed(rendition_filter):
+    """Checks a given rendition filter is allowed"""
+    allowed_filters = getattr(settings, "GRAPPLE_ALLOWED_IMAGE_FILTERS", [])
+    if not allowed_filters:
```

Comment:
This is a great observation. In my experience I've yet to find a Wagtail project that does not make use of renditions, but from a logical point of view if one sets this to `[]` it would makes sense to interpret it as "no image filters are allowed"

Reply:
changed a bit:
- any rendition is allowed if `GRAPPLE_ALLOWED_IMAGE_FILTERS` is not set, or if it is set but is not a list (probably overkill)
- then defer to checkign if filter is in allowed list. an empty list means no filters are allowed